playground/AzureContainerApps/AzureContainerApps.AppHost/api.module.bicep

```diff
@@ -3,10 +3,10 @@ param location string = resourceGroup().location
 
 param api_containerport string
 
+@secure()
 param storage_outputs_blobendpoint string
```

Comment:
Why's this parameter now considered "secure"? It is just a URI, right?

Reply:
We're currently marking everything that is a connection string as secure. We can be a bit more specialized but that can be a follow up.